free-2.md

```diff
@@ -58,11 +116,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+    I think the implementer would spend much more time on the language aspect
+    as opposed to the systems aspect. This is because there's not many
+    keywords to define. If I had to give a percentage, probably 80/20.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+    I think this isn't too big of an idea and isn't too ambitious, but I 
```

Comment:
I agree, you could limite the scope easily and just do it as a proof of concept aswell